### PR TITLE
use the updated upstream charts

### PR DESCRIPTION
**Pull request by SamClinckspoor**
Remove charts that are updated in upstream and install our charts in the infrastructure namespace.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,24 +26,24 @@ You can add this charts repo by:
...
 [`terraform-kubernetes/base`](https://github.com/skyscrapers/terraform-kubernetes/tree/master/base)
 module, then you can just start installing:
 
 ```console
-helm upgrade --install kube2iam skyscrapers/kube2iam --values helm-values.yaml
-helm upgrade --install kube-lego skyscrapers/kube-lego --values helm-values.yaml
-helm upgrade --install nginx-ingress skyscrapers/nginx-ingress --values helm-values.yaml
-helm upgrade --install external-dns skyscrapers/external-dns --values helm-values.yaml
-helm upgrade --install kubesignin skyscrapers/kubesignin --values helm-values.yaml
+helm upgrade --install kube2iam --namespace infrastructure --values helm-values.yaml
```
Comment (ringods) [inline README.md]: You removed the chart name to install in this and a few following lines. The word after `--install` is the helm release name.

File: README.md [hunk 2]
```diff
@@ -26,24 +26,24 @@ You can add this charts repo by:
...
+helm upgrade --install kube-lego --namespace infrastructure --values helm-values.yaml
+helm upgrade --install nginx-ingress --namespace infrastructure --values helm-values.yaml
```
Comment (MattiasGees) [inline README.md]: Should this go inside the infrastructure namespace?

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SamClinckspoor) [reply README.md]: I'm not sure, wi'll check

File: README.md (hunk 1 above)
Comment (SamClinckspoor) [reply README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -26,24 +26,24 @@ You can add this charts repo by:
...
 [`terraform-kubernetes/base`](https://github.com/skyscrapers/terraform-kubernetes/tree/master/base)
 module, then you can just start installing:
 
 ```console
-helm upgrade --install kube2iam skyscrapers/kube2iam --values helm-values.yaml
-helm upgrade --install kube-lego skyscrapers/kube-lego --values helm-values.yaml
-helm upgrade --install nginx-ingress skyscrapers/nginx-ingress --values helm-values.yaml
-helm upgrade --install external-dns skyscrapers/external-dns --values helm-values.yaml
-helm upgrade --install kubesignin skyscrapers/kubesignin --values helm-values.yaml
+helm upgrade --install kube2iam kube2iam --namespace infrastructure --values helm-values.yaml
```
Comment (ringods) [inline README.md]: Won't work yet. You need to pass a repo to install from. The stable Kubernetes repo is listed as `stable`:

```
$ helm repo list
NAME       	URL
stable     	https://kubernetes-charts.storage.googleapis.com
local      	http://127.0.0.1:8879/charts
skyscrapers	https://skyscrapers.github.io/charts
```
the chart names should read:
`stable/kube2iam` etc...

**Review by ringods — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (SamClinckspoor) [reply README.md]: isn't stable the default?

Comment (ringods) [reply README.md]: Don't know, but I would make it explicit.

Comment (SamClinckspoor) [reply README.md]: done

**Review by ringods — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (SamClinckspoor) [reply README.md]: it should work. from the nginx-ingress docs: `--watch-namespace string           Namespace to watch for Ingress. Default is to watch all namespaces`

**Review by venturel — APPROVED** [review]

**Pull request merged**